src/components/views/rooms/MessageComposer.tsx

```diff
@@ -255,7 +255,7 @@ interface IState {
     me?: RoomMember;
     narrowMode?: boolean;
     isMenuOpen: boolean;
-    showStickers: boolean;
+    stickersVisible: boolean;
```

Comment:
maybe `isStickerPickerOpen` to match `isMenuOpen` above

Reply:
I like it, thanks.

Reply:
Done in 0e9fd3e49f